examples/credential-registry/src/lib.rs

```diff
@@ -17,16 +17,21 @@
 //! - update the credential metadata;
```

Comment:
^ I see that this mentions the restore functionality. Didn't we agree not to have that?

Reply:
We agreed not to have "arbitrary" updates and keep the possibility to extend the credential lifecycle. So, it is currently possible for the issuer to restore a credential (flip back the revocation flag).